en/docs/chapter_graph/summary.md

```diff
@@ -1,31 +1,31 @@
 # Summary
 
-### Key review
-
-- A graph consists of vertices and edges and can be represented as a set comprising a group of vertices and a group of edges.
-- Compared to linear relationships (linked lists) and divide-and-conquer relationships (trees), network relationships (graphs) have a higher degree of freedom and are therefore more complex.
-- The edges of a directed graph have directionality, any vertex in a connected graph is reachable, and each edge in a weighted graph contains a weight variable.
-- Adjacency matrices use matrices to represent graphs, with each row (column) representing a vertex and matrix elements representing edges, using $1$ or $0$ to indicate the presence or absence of an edge between two vertices. Adjacency matrices are highly efficient for add, delete, find, and modify operations, but they consume more space.
-- Adjacency lists use multiple linked lists to represent graphs, with the $i^{th}$ list corresponding to vertex $i$, containing all its adjacent vertices. Adjacency lists save more space compared to adjacency matrices, but since it is necessary to traverse the list to find edges, their time efficiency is lower.
-- When the linked lists in the adjacency list are too long, they can be converted into red-black trees or hash tables to improve query efficiency.
-- From the perspective of algorithmic thinking, adjacency matrices embody the principle of "space for time," while adjacency lists embody "time for space."
-- Graphs can be used to model various real systems, such as social networks, subway routes, etc.
+### Key review points
```

Comment:
If we adopt this change, please update this title in `summary.md` of all the chapters.